src/app/content/Content.tsx

```diff
@@ -22,7 +23,14 @@ export function Content({ term }: ContentProps): JSX.Element {
   const { data, loading } = useGetCourse({ term, pid: searchParams.get('pid') || '' });
 
   return (
-    <Flex width={['container.md', 'container.lg', 'container.xl']} flexDirection="column" minH="100%">
+    <Flex width={['container.md', 'container.lg', 'container.xl']} flexDirection="column">
+      <Helmet>
+        {data?.subject && data?.code ? (
+          <title>{`${data?.subject} ${data?.code} · View Courses · clockwork`}</title>
```

Comment:
I feel this should be `Calendar` rather than `View Courses` given the buttons in the header.